zoo-css-challenge/style.css

```
@@ -178,6 +223,8 @@ footer {
 .membership {
   text-align: center;
   background-color: #fffffc;
+  max-width: 700px;
+  margin-left: 300px;
```

```suggestion
  margin: auto;
```
We can use `auto` which centres the element. `auto` evenly distributes the margin on the left and right